managed/src/main/resources/reference.conf

```diff
@@ -13,4 +13,12 @@ yb {
     # For how long do we let the task be in database after it has completed
     task_retention_duration = 120 days
   }
+
+  volumeInfo {
+      # Volume count needed for AWS instance types
```

Comment:
`# default volume count for aws instance types with EBS Only  storage info`

Reply:
Done